README.md

```diff
@@ -1,47 +1,153 @@
 # MN Bootstrap
 
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+
 > Distribution package for Dash Masternode installation
```

Comment:
Why are we formatting this line as a blockquote? We aren't quoting a person here, and the [Standard Readme spec states](https://github.com/RichardLitt/standard-readme/blob/master/spec.md#short-description) that the short description must not start with `>`.